<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerTractographyDisplayModule</class>
 <widget class="qSlicerWidget" name="qSlicerTractographyDisplayModule">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="SelectFiberBundleLabel">
     <property name="text">
      <string>FiberBundle:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="qMRMLNodeComboBox" name="FiberBundleNodeSelector">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="nodeTypes">
      <stringlist>
       <string>vtkMRMLFiberBundleNode</string>
      </stringlist>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Line</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="qSlicerTractographyDisplayWidget" name="LineDisplayWidget" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tube</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="qSlicerTractographyDisplayWidget" name="TubeDisplayWidget" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Glyph</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="qSlicerTractographyDisplayWidget" name="GlyphDisplayWidget" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="qSlicerTractographyDisplayGlyphWidget" name="GlyphPropertiesWidget" native="true"/>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerTractographyDisplayWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerTractographyDisplayWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerTractographyDisplayGlyphWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerTractographyDisplayGlyphWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerTractographyDisplayModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>LineDisplayWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerTractographyDisplayModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>FiberBundleNodeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FiberBundleNodeSelector</sender>
   <signal>currentNodeChanged(vtkMRMLNode*)</signal>
   <receiver>qSlicerTractographyDisplayModule</receiver>
   <slot>setFiberBundleNode(vtkMRMLNode*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerTractographyDisplayModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>GlyphDisplayWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerTractographyDisplayModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>TubeDisplayWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerTractographyDisplayModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>GlyphPropertiesWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
